src/controllers/user/controls/index.js

```diff
@@ -20,7 +21,7 @@ export default function (view) {
 
     view.addEventListener('viewshow', function () {
         view.querySelector('.smoothScrollContainer').classList.toggle('hide', !layoutManager.tv);
-
+        view.querySelector('.enableGamepadContainer').classList.toggle('hide', browser.edgeUwp);
```

Comment:
Just to separate hiding fields and restoring their values.
```suggestion
        view.querySelector('.enableGamepadContainer').classList.toggle('hide', browser.edgeUwp);

```
I would also sort them alphabetically.